Src/AutoMoq/MockVirtualMethodsCommand.cs

```diff
@@ -115,6 +116,14 @@ private static IEnumerable<MethodInfo> GetConfigurableMethods(Type type)
                               ? type.GetInterfaceMethods()
                               : type.GetMethods();
 
+            // Skip properties that have both getters and setters to not interfere
+            // with other post-processors in the chain which initialize them."
+            var getSetMethods = type.GetProperties()
+                                    .Where(p => p.GetGetMethod() != null &&
+                                                p.GetSetMethod() != null)
+                                    .SelectMany(p => new[] {p.GetGetMethod(), p.GetSetMethod()});
```

Comment:
Do we need both the `get` and `set` methods here? As far as I can see, the `CanBeConfigured` method allows to pick up the getters only (because the `set` methods are void), so this is redundant.
If that adds no value - it's better to keep the `get` methods only to make it simpler. Sorry that I overlooked that during my previous review 😊

P.S. If we agree to change this, feel free to use `git rebase -i` to edit previous commits and change this - it doesn't seem to add any value to the history 😉